### ISSUE TITLE
WCAG 2.1 level A violation in lightbox example on the `:target` selector page

### ISSUE THREAD

**Issue by robinmetral** [mdn/content#28587]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/:target

### What specific section or headline is this issue about?

Pure-CSS lightbox

### What information was incorrect, unhelpful, or incomplete?

The lightbox example contains at least one WCAG 2.1 level A violation

### What did you expect to see?

No WCAG 2.1 AA violations.

Whether examples need to comply with WCAG 2.1 AA could be discussed, since many simply illustrate whatever the MDN page is documenting, and do not resemble real-life use cases (see for instance the [first example on the `:target` selector page](https://developer.mozilla.org/en-US/docs/Web/CSS/:target#a_table_of_contents))

In this case however, since the lightbox is a functioning (albeit flawed) modal dialog, there is a risk that it ends up on live websites.

### Do you have any supporting links, references, or citations?

The lightbox fails at least one WCAG level A criterion: 2.1.1 keyboard: https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

Instead, such a lightbox should probably follow the behavior described in the [Dialog (modal) APG pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/)

### Do you have anything more you want to share?

I don't think it's worth trying to make the lightbox accessible, since it would likely not use the `:target` selector anymore. A CSS-only lightbox is not accessible and should not be built: I'd recommend removing the section.

I'm happy to open a PR for this, let me know.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/_colon_target`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/:target
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/_colon_target/index.md
* Last commit: https://github.com/mdn/content/commit/592f6ec42e54981b6573b58ec0343c9aa8cbbda8
* Document last modified: 2023-07-25T00:27:48.000Z

</details>

**Comment by sideshowbarker** [comment]
> I'm happy to open a PR for this, let me know.

Yes please do

**Comment by robinmetral** [comment]
Thanks—opened at #28623
